src/test/java/iq/RuntimeTests.java

```diff
@@ -94,7 +94,7 @@ public static void setupSparkSession() {
     public void testRuntimeIterators() throws Throwable {
         System.err.println(AnnotationsTestsBase.counter++ + " : " + this._testFile);
         TranslationVisitor visitor = new TranslationVisitor();
-        testAnnotations(this._testFile.getAbsolutePath(), visitor);
+        testAnnotations(this._testFile.getAbsolutePath());
```

Comment:
I just noticed that I missed a variable name with underscore here. Could you remove the underscore from the name?

Reply:
Done